spinn_utilities/configs/camel_case_config_parser.py

```diff
@@ -1,28 +1,32 @@
-from ConfigParser import RawConfigParser
-import distutils.util
+try:
+    from ConfigParser import RawConfigParser as parser
+except ImportError:
+    from configparser import RawConfigParser as parser
+import distutils.util as _du  # pylint: disable=import-error, no-name-in-module
 
 
-class CamelCaseConfigParser(RawConfigParser):
+class CamelCaseConfigParser(parser):
     # RawConfigParser is a classobj in Python 2.7, not a type (i.e., it
     # doesn't inherit from object), and so cannot be used with super().
+    __slots__ = ["_none_marker", "_read_files"]
 
     def optionxform(self, optionstr):
         lower = optionstr.lower()
         return lower.replace("_", "")
 
     def __init__(self, defaults=None, none_marker="None"):
-        RawConfigParser.__init__(self, defaults)
+        parser.__init__(self, defaults)
```

Comment:
Should this be super(...) or does this not work in Python 3?

Reply:
It doesn't work in Py2; `super(…)` requires the superclass inherit from `object`, which `RawConfigParser` doesn't in 2.7. There was much swearing when I found that out. 😡

Reply:
Fair enough - just wondered!